pkg/queue/stats/stats_reporter.go

```diff
@@ -191,3 +196,21 @@ func responseCodeClass(responseCode int) string {
 	// Get the hundred digit of the response code and concatenate "xx".
 	return strconv.Itoa(responseCode/100) + "xx"
 }
+
+// unregisterViews unregisters the views register in NewStatsReporter.
```

Comment:
considering this is a package private method to be used, I presume in package tests, this function can take Reporter pointer as a parameter and live completely in the _test file.

Reply:
👍

Reply:
Done.